src/components/mods/Level100MOptions.ts

```diff
@@ -0,0 +1,20 @@
+import { Level100Mod } from "./Level100Mod";
+import { MOptions } from "./MOptions";
+
+export class Level100MOptions extends MOptions {
```

Comment:
> `extends MOptions`

Once you remove `readIn` and separate storage from reading options (see other comments), do we really need a parent class? How about objects that just satisfy an interface?